contracts/teleporter/registry/tests/GetTeleporterMessengerTests.t.sol

```diff
@@ -10,14 +10,6 @@ import {ITeleporterMessenger} from "@teleporter/ITeleporterMessenger.sol";
 import {TeleporterMessenger} from "@teleporter/TeleporterMessenger.sol";
 
 abstract contract GetTeleporterMessengerTest is BaseTeleporterRegistryAppTest {
-    function testGetTeleporterMessengerBasic() public {
```

Comment:
was there a reason to move this

Reply:
seems like a lint with the new version, did it hit a lint rule?

Reply:
Yeah, the linter complained about `public view` functions being before `public` ones. `solc` was throwing warnings saying that this function could be marked `view`. I'm not sure what change prompted `solc` to start warning though.